### DISCUSSION TITLE
Pycaret 3.0 - Preprocess enable but plot_model(**, plot= 'feature') didn't show OHE properly?

### DISCUSSION THREAD

**Discussion by nutyfreshz** [pycaret/pycaret#2947] [Q&A]
As I'm a user that uses Pycaret 2.x version.
Normally I saw category features convert into OHE but now I use Pycaret 3.x and plot_model(plot = 'feature') I notice that category features didn't imply the same as before.

![image](https://user-images.githubusercontent.com/98257732/188839804-d3433cb7-0469-4678-93f3-8df0a73b138d.png)

Usually, they will show category as suffix value like:
![image](https://user-images.githubusercontent.com/98257732/188841651-6af988c1-3001-4576-bc2c-ce166b16548f.png)

Here is my setup config:
```
ls_model = setup(df_ls,
                 target = 'สถานะการโอน',
                 normalize = True,
                 normalize_method = 'minmax',
                 transformation = True,
                 transformation_method = 'yeo-johnson',
                 data_split_stratify = True,
                 session_id = 2022,
                 preprocess = True)
```
And set up dialog:
![image](https://user-images.githubusercontent.com/98257732/188840712-66e54f4a-67d5-4710-aa0e-57abbe3dc301.png)

Wrap-up questions on PyCaret 3.0
1.  Category features detected and convert into OHE ?
  1.1. If Yes, Why plot_model(plot ='feature') didn't show Category as OHE ?

Thank you.

**Comment by moezali1** [comment]
@nutyfreshz In the first example where you show Product Category, can you please show some sample dataset so I can assess what happens here.

**Reply by nutyfreshz** [reply]
@moezali1 Sorry for the late reply, Here is my df look like
![image](https://user-images.githubusercontent.com/98257732/189846539-63de9ad1-a20f-454c-9f6d-a261c47b845c.png)

**Reply by nutyfreshz** [reply]
All three features are marked as a category feature already but I didn't know why when I plot feature_importance_ and got that information.
It's should be shown like this:

Product Category_น้ำด่าง  -----------------------------------------------o
ชื่อสินค้า_น้ำด่าง 550 ml     ----------------------o
.
.
